src/app/models/Address.ts

```diff
@@ -0,0 +1,5 @@
+export interface Address {
+  Street: string;
```

Comment:
Good work @Genza999 , however, we should use camelCase for variable naming, its conventional in JS

Reply:
Thanks @CryceTruly , worked on this